visallo/visallo
# Fixes graph check issues

Testing Instructions:
- Create a global saved search
- Create a long running process
- Create user notification
- Create system notification
- Run graph check and verify no errors exist
